evolver/tests/test_device.py

```diff
@@ -145,9 +145,15 @@ def test_calibration_status(self, demo_evolver):
         assert isinstance(status["testsensor"].output_transformer, Status)
         assert status["testsensor"].ok
 
-    def test_abort(self, demo_evolver):
-        assert not demo_evolver.hardware["testeffector"].aborted
+    def test_abort(self, demo_evolver, monkeypatch):
+        test_effector = demo_evolver.hardware["testeffector"]
+        test_effector.aborted = False
+
+        def patched_abort(self):
+            self.aborted = True
```

Comment:
I think it would be better to still call the original ``abort`` here, to future-proof the test if something actually gets added to ``NoOpEffectorDriver.abort``, rather than circumvent the test, no?